styles.css

```diff
@@ -18,4 +22,579 @@ section {
 
 body {
   font-family: 'Lato', sans-serif;
```

Comment:
- [ ] As part of the project requirements please make sure to use all the appropriate fonts as you seem to be missing the "Cocogoose" font so kindly add this to your styles and revisit the Behance design to take a look as to where they're used.
- [ ] Although the 'Lato' font has been declared it's using the fallback font 'sans-serif' instead so please try to resolve this as well.
- Your design:
  ![image](https://github.com/balatstar/capstone1/assets/104968236/06124f4a-68d1-44b2-9060-7d18802dd2f9)
- Behance design:
  ![firefox_uWG5MkKUoF](https://github.com/balatstar/capstone1/assets/104968236/c8f021ff-4d1e-4838-bee2-a53173e06102)
- [ ] I'd also suggest you reduce some of the font-size on the mobile version of your design as they seem to be a bit larger when compared with the Behance design as shown above 👆